common/auth.py

```diff
@@ -0,0 +1,27 @@
+import os
+from common.constants import BASE_HEADERS
+
+# Ensure we have set the token before setting the constant for use elsewhere
+def set_token():
```

Comment:
Isn't this get_token()?

Reply:
I was using `set` because the function sets the value of the constant. Admittedly it's unintentionally a public method. Reading the code again I think it makes more sense to use these functions directly, rather than using the constant which has call the function anyway.

Reply:
Scratch the above, changing the function name to get is sufficient given the other uses in the constants file.

Reply:
Were you updating this one?